.editorconfig

```diff
@@ -472,7 +471,8 @@ ij_groovy_while_on_new_line = false
 ij_groovy_wrap_long_lines = false
 
 [{*.kt,*.kts}]
-disabled_rules=import-ordering
+# noinspection EditorConfigKeyCorrectness
+ktlint_disabled_rules=import-ordering,multiline-if-else,trailing-comma-on-call-site,trailing-comma-on-declaration-site
```

Comment:
Should we disable `import-ordering`?

Reply:
That was already disabled. Auto reformatting in Android Studio uses a different ordering than ktlint wants, so that results in some annoying errors. I couldn't figure out how to align them back then, so decided to disable that check